src/main/java/seedu/address/model/person/Person.java

```diff
@@ -2,13 +2,14 @@
 
 import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
 
-import java.util.ArrayList;
 import java.util.Collections;
 import java.util.HashSet;
 import java.util.Objects;
 import java.util.Set;
 
 import seedu.address.model.group.Group;
+//import seedu.address.model.group.GroupList;
```

Comment:
The model.group.GroupList is the list of all the available group. The used of static tells so. Should not appear in person. But however, we do need another list to show the list of group which the person is in. I think using set or arraylist is good.

Reply:
Once a person is assigned to a group, it will contain the group name, otherwise there is no link to group.

Reply:
Have explained more below.